bytes/bytes/config.py

```diff
@@ -136,6 +145,10 @@ class Settings(BaseSettings):
         None, description="OpenTelemetry endpoint", validation_alias="SPAN_EXPORT_GRPC_ENDPOINT"
     )
 
+    db_connection_pool_size: int = Field(
+        16, description="Database connection pool size", validation_alias="BYTES_DB_CONNECTION_POOL_SIZE"
+    )
```

Comment:
`validation_alias` is not necessary here and for mula because it will just use setting name and prepend the env_prefix.